packages/stream_core/lib/src/api/interceptors/headers_interceptor.dart

```diff
@@ -3,27 +3,19 @@ import 'package:dio/dio.dart';
 import '../system_environment_manager.dart';
 
 /// Interceptor that sets additional headers for all requests.
-class AdditionalHeadersInterceptor extends Interceptor {
-  /// Initialize a new [AdditionalHeadersInterceptor].
-  const AdditionalHeadersInterceptor(this._systemEnvironmentManager);
+class HeadersInterceptor extends Interceptor {
+  /// Initialize a new [HeadersInterceptor].
+  const HeadersInterceptor(this._systemEnvironmentManager);
 
   final SystemEnvironmentManager _systemEnvironmentManager;
 
-  /// Additional headers for all requests
-  static Map<String, Object?> additionalHeaders = {};
-
```

Comment:
I think I got this from chat, so devs can add more headers if they want. We don't need/allow this anymore, or do they need to make their own interceptor?

Reply:
I think this was originally there to set the x-stream-client headers from the flutter SDK's but now that we have a dedicated environment manager, i don't think we need it